From 31aed72218023764551dea2524e9d84277db4ef7 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Fri, 16 Aug 2019 13:52:12 +0800
Subject: [PATCH 046/245] Revert "drm/arm/mali-dp: Disable checking for
 required pixel clock rate"

commit ee3572c09c5e75f26bd0122f5830bb19565ef535 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Commit ded566c9facd160e force disable the pxlclk rate round checking, but it
not needed after added PIXEL clock provider driver, so revert that commit.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/gpu/drm/arm/malidp_crtc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/arm/malidp_crtc.c b/drivers/gpu/drm/arm/malidp_crtc.c
index 7abe70174f87..fcc62bc60f6a 100644
--- a/drivers/gpu/drm/arm/malidp_crtc.c
+++ b/drivers/gpu/drm/arm/malidp_crtc.c
@@ -36,13 +36,11 @@ static enum drm_mode_status malidp_crtc_mode_valid(struct drm_crtc *crtc,
 
 	if (req_rate) {
 		rate = clk_round_rate(hwdev->pxlclk, req_rate);
-#ifndef CONFIG_ARCH_LAYERSCAPE
 		if (rate != req_rate) {
 			DRM_DEBUG_DRIVER("pxlclk doesn't support %ld Hz\n",
 					 req_rate);
 			return MODE_NOCLOCK;
 		}
-#endif
 	}
 
 	return MODE_OK;
-- 
2.17.1

